modules/installation-configuration-parameters.adoc

```diff
@@ -320,6 +324,28 @@ with an Azure cluster.
 ====
 endif::azure[]
 
+
+ifdef::gcp[]
+.Additional Google Cloud Platform (GCP) parameters
+[cols=".^2,.^3a,.^3a",options="header"]
+|====
+|Parameter|Description|Values
+
+|`platform.gcp.network`
+|The name of the existing VPC that you want to deploy your cluster to.
+|String.
+
+|`platform.gcp.controlPlaneSubnet`
+|The name of the existing subnet in your VPC that you want to deploy your control plane machines to.
+|Valid CIDR, for example `10.0.0.0/16`.
```

Comment:
@kalexand-rh The value of platform.gcp.controlPlaneSubnet is supposed to be the subnet name instead of CIDR.